org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/AbstractADWindowContent.java

```diff
@@ -877,9 +879,10 @@ private void initialQuery(final MQuery query, GridTab mTab, final Callback<MQuer
         	m_findCancelled = false;
         	m_findCreateNew = false;
             GridField[] findFields = mTab.getFields();
-            FindWindow findWindow = new FindWindow(curWindowNo, mTab.getTabNo(),
-                    mTab.getName(), mTab.getAD_Table_ID(), mTab.getTableName(),
-                    where.toString(), findFields, 10, mTab.getAD_Tab_ID(), this); // no query below 10
+            
+            IFindWindowFactory findWindowFactory = Service.locator().locate(IFindWindowFactory.class).getService();
+            FindWindow findWindow = findWindowFactory.getInstance(curWindowNo, 0, title, mTab.getAD_Table_ID(), mTab.getTableName(), where.toString(), findFields, 10, mTab.getAD_Tab_ID(), this);
```

Comment:
Instead of repeating this, this should be added to` org.adempiere.webui.Extensions` instead.